baldur/src/main/webapp/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
```

Comment:
check luskan for title and doc header style and be consistent with that

Reply:
Updated.